.circleci/config.yml

```diff
@@ -299,9 +299,6 @@ jobs:
       - trust-github-key
       - update-carthage-integration-commit
       # Carthage
-      - restore_cache:
-          keys: 
-            - carthage-cache-{{ checksum "Cartfile.resolved" }}
```

Comment:
this hasn't worked for a while, I think we already had removed it from some branches for releases but never `main` I guess